### ISSUE TITLE
Missing Customer Object

### ISSUE THREAD

**Issue by luklus** [chec/commerce.js#129]
```js
const commercejs = new CommerceSDK(commercejsAPIKEY, isDevelopmentEnvironment)

console.log(commercejs)
```
 I get result:

```js
cart: a {commerce: a, cartId: null}
categories: a {commerce: a}
checkout: Checkout {commerce: a}
consoleHelper: ƒ ()
merchants: a {commerce: a}
options: {version: "v1", url: "https://api.chec.io/", publicKey: "pk_xxxx", debug: true, eventCallback: ƒ}
products: a {commerce: a}
services: a {commerce: a}
storage: a {commerce: a}
```

There is no Customer Object, so I can't login or use others options ex. isLoggedIn()

**Comment by ricardodesimas** [comment]
Hi @luklus, the Customer feature is not available in the CommerceJS at this moment, you could access the API endpoints with your API key, check out the Customer API docs here: https://commercejs.com/docs/api/#customers

**Comment by robbieaverill** [comment]
Hi @luklus, the customer features in Commerce.js were available in v2.3.0-beta2, I've just published them under v2.3.0. Cheers!
